src/components/node-list/node-list-items.test.js

```diff
@@ -120,15 +120,14 @@ describe('node-list-selectors', () => {
   describe('getSections', () => {
     const sections = getSections({ flags: { modularpipeline: false } });
 
-    const groupType = 
-      expect.objectContaining({
-        name: expect.any(String),
-        types: expect.any(Array),
-      });
+    const groupType = expect.objectContaining({
```

Comment:
Similar to the above comment, I understand that these changes could be due to some prettier settings on update, but given that there are no changes introduced to the code and it is not related to this PR, please avoid commiting these type of unrelated changes in the PR to contaminate the clarity in the PR.